docker-compose.yml

```diff
@@ -101,6 +104,7 @@ services:
     depends_on:
       - rabbitmq
       - registry
+      - memcached
```

Comment:
Can you configure the `DevelopmentConfig` class to connect to this memached instance but override it in `TestConfiguration` back to `dogpile.cache.null`?